src/ruleLoader.ts

```diff
@@ -43,11 +45,18 @@ export function loadRules(ruleConfiguration: {[name: string]: any},
                 const ruleSpecificList = (ruleName in enableDisableRuleMap ? enableDisableRuleMap[ruleName] : []);
                 const disabledIntervals = buildDisabledIntervalsFromSwitches(ruleSpecificList, allList);
                 rules.push(new Rule(ruleName, ruleValue, disabledIntervals));
+            } else {
```

Comment:
let's change this conditional block to read better (prefer the positive form of conditional expressions first):

```ts
if (Rule == null) {
  notFoundRules.push(ruleName);
} else {
  ...
}
```